src/sentry/search/events/fields.py

```diff
@@ -2483,8 +2569,8 @@ def resolve_function(self, function: str, match: Optional[Match[str]] = None) ->
 
         arguments = snql_function.format_as_arguments(name, arguments, self.params)
         for arg in snql_function.args:
-            if isinstance(arg, SnQLColumn):
-                arguments[arg.name] = self.resolve_field(arguments[arg.name])
+            if type(arg) in [SnQLNumericColumnNoLookup, SnQLColumn]:
+                arguments[arg.name] = self.resolve_column(arguments[arg.name])
```

Comment:
is this change necessary? can it remain `self.resolve_field`?

Reply:
Hey Shruthi, looks like there was a recent change to `resolve_field`. `resolve_function` and `resolve_field_alias` were moved out of `resolve_field` and into the new function `resolve_column` which also calls `resolve_field`. Since we would need to support the measurement aliases for percentile, I think we're required to use `resolve_column` for at least `SnQLNumericColumnNoLookup`. Maybe @wmak can comment on this, is this correct?

Reply:
can confirm, I've changed resolve_field to mean specifically that we're resolving a field (ie. we know what type the alias is already) compared to resolve_column where we're not usre if its an alias, column or field